src/components/streamSelect/StreamSelect.css

```diff
@@ -0,0 +1,51 @@
+.options-bar {
+  height: 35px;
+  background-color: white;
```

Comment:
Looks bad to me. Can't/shouldn't we use something more in line with the rest of the color scheme used?